### PR TITLE
First drop of simple mount_syscall tool for integration-testing purposes

### PR DESCRIPTION
**Pull request by rodnymolina**
This code is just a placeholder at the moment. It's not being compiled nor executed by anyone/anything. That is expected to happen in the next few days as part of some integration-testcases that will be written.

Signed-off-by: Rodny Molina <rmolina@nestybox.com>

### CODE REVIEW

File: tests/syscall/mount_syscall.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+func usage() {
+	fmt.Printf("\nUsage: mount_syscall <bind | ro-remount | rw-remount> <source> <target>\n\n")
+}
+
+func main() {
+	args := os.Args[1:]
+
+	if len(args) != 3 {
+		fmt.Printf("\nNumber of attributes received: %d, expected: %d\n", len(args), 3)
```
Comment (ctalledo) [inline tests/syscall/mount_syscall.go:43]: nit: attributes -> arguments

File: tests/syscall/mount_syscall.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+	"golang.org/x/sys/unix"
+)
+
+func do_bind_mount(s, t string) {
+
+	err := unix.Mount(s, t, "", unix.MS_BIND, "")
+	if err != nil {
+		fmt.Printf("bind-mount error received %v:", err)
+	}
```
Comment (ctalledo) [inline tests/syscall/mount_syscall.go:16]: We should return an error here, and have main() exit with an error code (so bats can catch the failures).

**Review by ctalledo — CHANGES_REQUESTED** [review]
Some minor comments.

File: tests/syscall/mount_syscall.go (hunk 2 above)
Comment (rodnymolina) [reply tests/syscall/mount_syscall.go:16]: yep, done

**Review by ctalledo — APPROVED** [review]
LG!

**Pull request merged**